charts/pega/README.md

```diff
@@ -86,10 +86,10 @@ jdbc:
   driverClass: org.postgresql.Driver
 ```
 
-#### (Optional) Support for providing DB credentials using External Secrets Operator
+#### (Optional) Support for providing credentials/certificates using External Secrets Operator
 
-To avoid directly entering your passwords as plain text in your Helm charts, Pega supports Kubernetes secrets to secure credentials and related information. 
-Use secrets to represent credentials for your database, Docker registry, or any other token or key that you need to pass to a deployed application. Your secrets can be stored in any secrets manager provider. 
+To avoid directly entering your confidential content such as passwords, certificates as plain text in your Helm charts, Pega supports Kubernetes secrets to secure credentials and related information. 
```

Comment:
small tweaks>
To avoid directly entering your confidential content such as passwords, certificates as plain text in your Helm charts, Pega supports Kubernetes secrets to secure credentials and related information. 
Use secrets to represent credentials for your database, Docker registry, SSL certificates or any other token or key that you need to pass to a deployed application. Your secrets can be stored in any secrets manager provider. 
to
To avoid directly entering your confidential content in your Helm charts such as passwords or certificates in plain text, Pega supports Kubernetes secrets to secure credentials and related information. 
Use secrets to represent credentials for your database, Docker registry, SSL certificates, or any other token or key that you need to pass to a deployed application. Your secrets can be stored in any secrets manager provider. 

BTW @smootherbug - is ANY secrets manager valid or do we have a list that we support?

Reply:
we use a framework called external secrets provider, so it supports certain secret managers, we support those